coordinator/gscoordinator/local_launcher.py

```diff
@@ -232,7 +232,8 @@ def create_interactive_instance(
             str(num_workers),  # server size
             str(self._interactive_port),  # executor port
             str(self._interactive_port + 1),  # executor rpc port
-            str(self._interactive_port + 2 * num_workers),  # frontend port
+            str(self._interactive_port + 2 * num_workers),  # frontend gremlin port
+            str(self._interactive_port + 2 * num_workers + 1),  # frontend cypher port
```

Comment:
Try launch multiple interactive instance, and see if the port will conflict?
I mean, the line 240 may need to change.